migrations/9_SCHEME-REP-Auction_Gen.js

```diff
@@ -60,9 +63,11 @@ module.exports = async function (deployer) {
   console.log('  - Staking token address (GEN):', genToken.address)
   console.log('  - wallet address (DxAvatar.address): ', walletAddress)
 
+
+
   await dxGenAuction4Rep.initialize(
     dxAvatar.address,
-    reputationReward,
+    reputationReward/* / numberOfGenAuctions*/,
```

Comment:
Why is `/numberOfGenAuctions` commented out?
Would you like to pick commit [db06adb7046a8015e2307f039caf622928a75fa2](https://github.com/gnosis/dx-daostack/pull/41/commits/db06adb7046a8015e2307f039caf622928a75fa2)?